toto/toto-run.py

```diff
@@ -132,9 +132,9 @@ def main():
   product_list = []
 
   if args.materials:
-    material_list = args.materials.split(",")
+    material_list = args.materials
   if args.products:
-    product_list = args.products.split(",")
+    product_list = args.products
 
```

Comment:
Not at all, I wanted to not pollute the diff (so that if we need to add any validation to it we donn't have problems). I could do so if you think this is better.